src/couch_jobs/src/couch_jobs.erl

```diff
@@ -297,9 +297,17 @@ accept_loop(Type, NoSched, MaxSchedTime, Timeout) ->
     TxFun =  fun(JTx) ->
         couch_jobs_fdb:accept(JTx, Type, MaxSchedTime, NoSched)
     end,
-    case couch_jobs_fdb:tx(couch_jobs_fdb:get_jtx(), TxFun) of
+    AcceptResult = try
+        couch_jobs_fdb:tx(couch_jobs_fdb:get_jtx(), TxFun)
+    catch
+        error:{erlfdb_error, Err} when Err =:= 1020 orelse Err =:= 1031 ->
+            retry
+    end,
+    case AcceptResult of
         {ok, Job, Data} ->
             {ok, Job, Data};
+        retry ->
+            accept_loop(Type, NoSched, MaxSchedTime, Timeout);
```

Comment:
Am I reading this correctly that this will retry loop forever in the event it continues to timeout? Is there something else that will time this out? Or do we want to just retry indefinitely for jobs?

Reply:
We are ok retrying it forever in a loop in case of transaction timeouts. We will exit the loop with `{error, not_found}` in case the user specifies a `Timeout` option but we don't want the global (default) transaction settings to affect indexers waiting to accept jobs